_dashboards/im-dashboards/notifications.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: Notification settings
+parent: Index management in Dashboards
+nav_order: 60
+---
+
+# Notification settings
+
+You can configure default notification settings for index operations globally on the **Notification settings** page. If you need to override any of the settings, you can also configure notification settings for individual index operations.
+
+## Configuring default notification settings
+
+On the **Notification settings** page, you can configure the default notification settings for the following index operations that may take longer time to complete:
+
+- Open
+- Reindex
+- Split
+- Shrink
+- Clone
+- Force merge
+
+To get started, from the top menu, select **OpenSearch Plugins** > **Index Management**. Under **Index Management**, select **Notification settings**.
+
+You can choose to be notified when the operation has completed or failed. Additionally, you can select the notification channels for this notification, as shown in the following image:
+
+![Default notification settings]({{site.url}}{{site.baseurl}}/images/admin-ui-index/notifications.png)
+
+If you don't have permissions to view notification settings, you will not be able to view the default settings. 
+{: .note}
+
+## Configuring notification settings for an individual operation
+
+You can view or override default notification settings when you perform an indexing operation. For example, if you want to configure additional notification for a reindex operation, perform the following steps:
```

Comment:
@kolchfa-aws FYI: In an individual operation flow, it will only allow users to add additional notification settings, it won't remove or overwrite what's being set by the default notification settings.